extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -394,7 +394,7 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 			}
 
 		} catch (error) {
-			console.log(error);
+			logError(TelemetryViews.SkuRecommendationWizard, 'GetSkuRecommendationFailed', error);
```

Comment:
sendErrorEvent takes in a string for the errorCode as the 3rd parameter - not an error object 

https://github.com/microsoft/ads-extension-telemetry/blob/main/src/telemetryReporter.ts#L210

So I don't think this is going to do what you want.

Reply:
@Charles-Gagnon . for this release we are okay with just getting how many times error happened during a workflow. Till we figure out how to get an actual error (without PII).

Reply:
Sounds good, just remove the error parameter then and this should be good

Reply:
export function logError(telemetryView: TelemetryViews, err: string, error: any): void {
	console.log(error);
	TelemetryReporter.sendErrorEvent(telemetryView, err);
}
error parameter is used for writing to console.